.github/workflows/tests.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
 
       - name: build tests
         run: |
-          cargo nextest run --workspace ${{ env.FEATURES }} --lib --bins --tests --no-run
+          cargo nextest run --workspace ${{ env.FEATURES }} --lib --bins --tests --profile ci --no-run
```

Comment:
I don't think this works the way you expected. IIRC this shifts the nextest profile which is a different thing from the cargo run build profiles. 

If you look at `.config/nextest.toml` you will see that this specifies an execution setup but is not related to the build artifacts in any way. Also the reason running didn't trigger a rebuild.

Reply:
eh.  TIL.  Reverted